.circleci/config.yml

```diff
@@ -3,6 +3,8 @@ jobs:
   build:
     docker:
       - image: circleci/node:10
+      - image: mosic/swarm
+        command: [./start_swarm_node.sh, tmp]
```

Comment:
Will the temporary directory be created for each build? We need to make sure builds don't get data left from previous tests, otherwise we could have false-positive integration tests.

Reply:
As I understood from CircleCI docs, docker images aren't cached between test runs, so a new temporary directory will be used every time in our case. Correct me if I'm wrong though.